rocky/reports/views/generate_report.py

```diff
@@ -114,7 +114,7 @@ def post(self, request, *args, **kwargs):
             messages.error(request, _("You do not have the required permissions to enable plugins."))
             return PostRedirect(self.get_previous())
 
-        client = get_katalogus(self.organization.code)
+        client = get_katalogus(self.organization_member)
         for selected_plugin in selected_plugins:
             try:
                 client.enable_boefje_by_id(selected_plugin)
```

Comment:
I'd expect this HTTPError to be caught in the client, and instead a more specialized KatalogusClientHttpError subclassed from the KatalogusError or something to be raised.